packages/perseus/src/widgets/interactive-graphs/backgrounds/axis-ticks.tsx

```diff
@@ -47,34 +36,53 @@ const YGridTick = ({y, graphInfo}: {y: number; graphInfo: GraphDimensions}) => {
     const x2 = xPosition + tickSize / 2;
     const y2 = yPosition;
 
+    // Adjust the y position of the x-axis labels based on
+    // whether the x-axis is above, within, or below the graph
+    const xAdjustment =
+        range[X][MAX] <= 0 ? tickLabelSize * 1.5 : -tickLabelSize * 1.1;
+    const xPositionText = xPosition + xAdjustment;
```

Comment:
I'm not a big fan of using 1.1 here, but I found it was the best value to visually line up the labels for our most used gridStep/ranges